src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -619,6 +619,7 @@ std::vector<std::string> disabledTestPatterns() {
     if (!ov::with_cpu_x86_avx512_core_amx_bf16() && !ov::with_cpu_x86_bfloat16() && !CPUTestUtils::with_cpu_x86_avx2_vnni_2()) {
         retVector.emplace_back(R"(.*smoke_Snippets_EnforcePrecision_bf16.*)");
         retVector.emplace_back(R"(.*smoke_Snippets_MHAWOTransposeEnforceBF16.*)");
+        retVector.emplace_back(R"(.*smoke_Snippets_FullyConnected_EnforceBF16.*)");
```

Comment:
Do we need to skip it here? I guess if we add `CPUTestUtils::cpu_bf16_plugin_config` the test will be not instantiated in case CPU does not support bf16

Reply:
Yes, you're right, I confused it with `filterCPUInfoForDevice`